lib/elixir/lib/kernel.ex

```diff
@@ -1384,6 +1384,16 @@ defmodule Kernel do
       iex> [1, 2, 3, 2, 1] -- [1, 2, 2]
       [3, 1]
 
+  It is important to note that `--/2` is right associative meaning:
```

Comment:
```suggestion
  It is important to note that the `--/2` operator has a *right to left* associativity:
```

Reply:
I personally find "right associative" clearer, which seems to be correct usage from the wikipedia page: https://en.wikipedia.org/wiki/Operator_associativity

Reply:
```suggestion
The `--/2` operator is right associative, meaning:
```